app/serializers/api/admin/enterprise_role_serializer.rb

```diff
@@ -1,5 +1,5 @@
 class Api::Admin::EnterpriseRoleSerializer < ActiveModel::Serializer
-  attributes :id, :user_id, :enterprise_id, :user_email, :enterprise_name
+  attributes :id, :user_id, :enterprise_id, :user_email, :enterprise_name, :receives_notifications
```

Comment:
`receives_notifications` doesn't seem to be used anywhere, so we probably don't need to serialise it.

Reply:
It's being used in angular controllers.

Reply:
I may be going crazy but grepping for `receives_notifications` gives me nothing anywhere in the JS.